manifest.webapp

```diff
@@ -66,5 +70,13 @@
       "type": ["io.cozy.contacts"],
       "href": "/services"
     }
-  ]
+  ],
+  "services": {
+    "normalizeContacts": {
```

Comment:
Not a big fan of `normalizeContacts` since to me the service is not normalizing any contact. 

Does something like: `keepIndexFullNameAndDisplayNameUpToDate` more relevant?